docs/synthesis.md

```diff
@@ -56,9 +69,10 @@ synthesis:
 
 | **Name** | **Type** | **Description**  |
 | -------- | -------- | ---------------- |
-| attribute | String  | Required. The name of the attribute to match in the data point. If only this field is defined the attribute must exist in the data point. |
-| value    | String   | Optional. Can't be mixed with `prefix`. The exact value the attribute must contain in order to match the data point. |
-| prefix | String | Optional. Can't be mixed with `value`. The attribute must start with this value. |
+| attribute | String  | Required. The name of the attribute to match in the data point. |
+| value    | String   | Optional. Can't be mixed with `prefix` or `present`. The exact value the attribute must contain in order to match the data point. |
+| prefix | String | Optional. Can't be mixed with `value` or `present`. The attribute must start with this value. |
+| present | Boolean | Optional. Defaults to true when no other condition is given. Can't be mixed with `value` or `prefix`. The attribute must be present or absent. |
```

Comment:
I would add a description of what the attributes means. A small description at the beginning. And then all the info regarding default value, can't be mixed etc. Maybe it's the last sentence what you meant. What about something at the beginning similar to what you put on the PR description: "Defines whether the condition requires the attribute to be present in the telemetry or not"

Reply:
Didn't thought a lot of it was just following the current format:

So the explanation is the "The attribute must be present or absent." part but I do agree that there's too much info in the description field. I will do a PR to order this a bit better.